packages/server/cloud/nlp/package.json

```diff
@@ -13,24 +12,28 @@
     "start": "functions-framework --source=build/src/ --target=nlp --port=7869",
     "watch": "concurrently \"tsc -w\" \"nodemon --watch ./build/ --exec npm run start\"",
     "test": "echo \"Error: no test specified\" && exit 1",
-    "lint": "gts lint",
-    "fix": "gts fix",
-    "clean": "gts clean",
-    "compile": "npm run lint && npm run clean && tsc",
-    "deploy": "npm run compile && gcloud functions deploy nlp --entry-point nlp --project adaptive-trader --trigger-http --allow-unauthenticated --runtime nodejs14"
+    "format:check": "prettier . --check",
```

Comment:
we don't really need the opinionated restrictions of `gts` on this very simple project .. just fall back to simple and distinct `eslint`, `prettier` and `tsc`